frontend/src/modules/Maintenance/views/MaintenanceViewer.js

```diff
@@ -0,0 +1,438 @@
+import {
```

Comment:
MInor: MaintenanceViewer is not a view, it is a component of a view, so we should have something like `components/MaintenanceViewer.js` instead

Reply:
So the reason I kept it as a view is because we are saying `Maintenance` as a module and this module provides as view - which works as a component - for the `AdministrativeView.js` . If MaintenanceViewer.js had a component inside of it then I was planning to add them in `component/` folder. 

I don't have much preference with the two. but I can refactor it to say as a component